types/rdf-dataset-ext/toStream.d.ts

```diff
@@ -1,3 +1,9 @@
-import { toStream } from '.';
+import { BaseQuad, DatasetCore, Quad, Stream } from 'rdf-js';
+import { Readable } from 'stream';
```

Comment:
Reading https://github.com/nodejs/readable-stream/issues/462#issue-1001117853 it's probably safer to go for

```suggestion
import { Readable } from 'readable-stream';
```